tests/test_session.py

```diff
@@ -218,6 +219,7 @@ class TestCreationApi:
         assert not result.age
         await app.close()
 
+    @pytest.mark.skipif(pytest.config.getoption('provider') == 'dse', reason='DSE')
     @pytest.mark.asyncio
```

Comment:
It's temporary -- because of this bug I found: http://stackoverflow.com/questions/40515447/is-dropping-an-edge-property-expected-to-drop-its-edge